### PR TITLE
Generate UUID value methods

### LINKED ISSUE #341: `UuidValue` API is puzzling
The interface now provides _instance_ methods `generate()` and `of(String)`. So, to use them we need something like:

```java
   CommandId id = CommandId.getDefaultInstance().generate();
```
This is not very convenient, and is not what the user expects. The generated code should allow doing this:

```java
   CommandId id = CommandId.generate();
```

Also, the factory method `of(String)` should probably check that the passed value is a valid UUID value string. Currently it does not.

### PR DESCRIPTION
**Pull request by yuri-sergiichuk**
This PR fixes #341 and introduces the default method generation configuration used for the `UuidValue` message.

The `UuidMethodFactory` creates two static methods:
1. `generate()` — creates a new instance of the UUID message with a randomly-generated `uuid` value.
1. `of(String uuid)` — creates a new instance of the UUID message with a supplied `uuid` value.

### CODE REVIEW

File: base/src/test/java/io/spine/tools/method/MethodFactoryTest.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+        public TestMethodFactory() {
+        }
+
+        @Override
+        public List<GeneratedMethod> createFor(MessageType messageType) {
+            return ImmutableList.of();
+        }
+    }
+
```
Comment (armiol) [inline base/src/test/java/io/spine/tools/method/MethodFactoryTest.java]: Please kill.

File: base/src/test/proto/spine/test/code/proto/test_uuid_messages.proto [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+option (type_url_prefix) = "type.spine.io";
+option (SPI_all) = true;
+option java_package = "io.spine.test.code.proto.uuid";
+option java_multiple_files = true;
+option java_outer_classname = "MttUuidMessagesProto";
+
+// This file defines test environment for `MessageTypeTest.java`.
+
+// A sample uuid message to verify detecting that a message is a UuidValue from `MessageType`.
```
Comment (armiol) [inline base/src/test/proto/spine/test/code/proto/test_uuid_messages.proto]: `UUID` and `UuidValue`.

File: base/src/test/proto/spine/test/tools/method/uuid/uuid_messages.proto [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+package spine.test.tools.method.uuid;
+
+import "spine/options.proto";
+
+option (type_url_prefix) = "type.spine.io";
+option java_package="io.spine.test.tools.method.uuid";
+option java_multiple_files = true;
+option java_outer_classname = "UuidMessageProto";
+
+
```
Comment (armiol) [inline base/src/test/proto/spine/test/tools/method/uuid/uuid_messages.proto]: Please kill this line.

**Comment by yuri-sergiichuk** [comment]
@armiol PTAL again.

File: base/src/main/java/io/spine/base/UuidValue.java [hunk 4]
```diff
@@ -46,8 +44,7 @@
  * @param <I>
  *         the type of the message
  */
-@SuppressWarnings({"unchecked" /* Class cast correctness guaranteed by the interface contract. */,
-        "InterfaceNeverImplemented" /* Used by the Protobuf Compiler plugin. */})
+@SuppressWarnings("InterfaceNeverImplemented")/* Used by the Protobuf Compiler plugin. */
```
Comment (armiol) [inline base/src/main/java/io/spine/base/UuidValue.java]: Please have at least one space before ‘/*’

**Pull request merged**